src/amplitude_experiment/local/client.py

```diff
@@ -85,10 +113,11 @@ def __setup_connection_pool(self):
 
     def close(self) -> None:
         """
-        Close resource like connection pool with client
```

Comment:
Node using `stop`. Should we keep it aligned?

Reply:
I pick `close` earlier to align with remote client. I can also rename if we feel it is better to align with node

Reply:
Just updated, let's try rename for now